util/util.go

```diff
@@ -3,8 +3,10 @@ package util
 import (
 	"fmt"
 	"strconv"
+	"time"
 
 	"github.com/pkg/errors"
+	starlibtime "github.com/qri-io/starlib/time"
```

Comment:
Sad that we need this import coupling, but it makes sense given how low a `Time` data structure should live in the stack

Reply:
@b5 agreed! I couldn't find an alternative